libraries/chain/include/eosio/chain/fork_database.hpp

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include <fc/io/cfile.hpp>
```

Comment:
Move below other includes. Consider forward declare so including the header can be avoided.

Reply:
Thanks, switched to forward declaration.